### PR TITLE
Prepare release 2.0.0

### PR DESCRIPTION
**Pull request by hramezani**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
 
     services:
       postgres:
-        image: postgres:10
```
Comment (Linkid) [inline .github/workflows/test.yml:16]: Why did you updated the version ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,8 +1,12 @@
 # Changes
 
+## 2.0.0 (2022-05-04)
+
+### Final
```
Comment (Linkid) [inline CHANGELOG.md:5]: I think this title can be removed as there was no alpha-like version this time ;).

**Review by Linkid — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (hramezani) [reply .github/workflows/test.yml:16]: Django added minimum database version checks in https://github.com/django/django/commit/9ac3ef59f9538cfb520e3607af743532434d1755
Django main requires at least Postgres 11
https://github.com/django/django/blob/21d8ea4eb3e22d458515b5278c8cd3a15b069799/django/db/backends/postgresql/features.py#L9

File: CHANGELOG.md (hunk 2 above)
Comment (hramezani) [reply CHANGELOG.md]: Removed!

**Comment by hramezani** [comment]
@Linkid I've addressed your comment and updated the release date to next Monday(2022-05-09). I will release the new version next week.

**Review by Linkid — APPROVED** [review]

**Pull request merged**